src/web3/networkConfig.ts

```diff
@@ -94,6 +94,7 @@ interface NetworkConfig {
   arbAavePoolV3ContractAddress: Address
   arbAaveIncentivesV3ContractAddress: Address
   aaveArbUsdcTokenId: string
+  valoraRpcUrl: Record<Network.Arbitrum, string>
```

Comment:
probably can be a single base url from which we can compute per n/w if we go with the url format as below, will clean in a follow up once we confirm this pattern works